### PR TITLE
Project base and initial models

### LINKED ISSUE #3: Project base and initial models

### PR DESCRIPTION
**Pull request by tuomas777**
Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+* Set the `DEBUG` environment variable to `1`.
+* Run `python manage.py migrate`
+* Run `python manage.py runserver 0:8000`
+
+## Running tests
+
+* Set the `DEBUG` environment variable to `1`.
+* Run `pytest`.
+>>>>>>> 7e41476... Project base
```
Comment (igordavydsson) [inline README.md]: A bit of git stuff there

Comment (igordavydsson) [reply README.md]: Ah, deleted in the next commit, my bad :slightly_smiling_face:

Comment (tuomas777) [reply README.md]: Nah, it shouldn't have been like that, fixed it properly now :)

**Review by igordavydsson — APPROVED** [review]
Do not see anything alarming. :shipit: 

BTW: what Python version are we gonna use there? are we gonna strive for python <3.3 support with those `__init__.py`?

**Comment by tuomas777** [comment]
Not sure which Python version we want to support, that is more @tituomin's territory, but I'm quite sure it won't be < 3.3, so those empty `__init__.py` could and probably should be removed, good point.

**Comment by tuomas777** [comment]
Removed empty `__init__.py` files from all packages except for migrations which seems to need one.

**Comment by igordavydsson** [comment]
Nice

**Review by tituomin — APPROVED** [review]

**Pull request merged**